.github/workflows/publish.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
         set -eux
         echo $CHARMSTORE_CREDENTIAL > ~/.go-cookies
         IMAGE=$(yq eval '.resources.oci-image.upstream-source' metadata.yaml)
-        charmcraft build
+        tox -e build
```

Comment:
It seems a little strange to me to involve tox in the build process. I understand why you did it, but since it's just one extra line to include the `local_sdi` step, maybe we should just inline it?

Reply:
I chose tox to simplify the installing the SDI python package step and always having the same comand/environment.